src/context/full.tsx

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+export const FullPageData = (props: { title: string }) => {
```

Comment:
Not sure how you use this. But if this is going to be a hook it should start with use in it's name.